src/main/java/edu/hawaii/its/api/util/JsonUtil.java

```diff
@@ -49,7 +65,6 @@ public static void prettyPrint(Object object) {
             String json = new ObjectMapper()
                     .writerWithDefaultPrettyPrinter()
                     .writeValueAsString(object);
-            System.out.println(json);
```

Comment:
We need this for the pretty print.

Reply:
Good catch, that was what made some of the builds fail. Thank you!